src/components/code-snippet/__tests__/code-snippet-test-cases.js

```diff
@@ -64,4 +64,26 @@ testCases.withHelpers = {
   }
 };
 
+testCases.maxHeight = {
+  component: CodeSnippet,
+  description:
+    'CodeSnippet with edit buttons, maxHeight, editClasses, and editStyles.',
+  props: {
+    code: fullHtml,
+    maxHeight: 150,
+    editClasses: 'absolute-mm right mb3 mt-neg36-mm',
```

Comment:
So is the idea here that I would add `editClasses='absolute-mm right mb3 mt-neg36-mm'` to the `FullDemoCode` component (i.e. the buttons will not auto reposition if `maxHeight` is used)?

Reply:
@colleenmcginnis ahh good point! I got the design backwards.Seems like when `maxHeight` is set, then we should automatically move the EditButtons (if enabled) above the CodeSnippet. I'll update.

Reply:
@colleenmcginnis ok updated!